msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -569,6 +569,7 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 			_CompileEntitlements;
 			_CompileAppManifest;
 			_ResolveAppExtensionReferences;
+			_ExtendAppExtensionReferences;
```

Comment:
You could probably remove `_ResolveAppExtensionReferences` here, since `_ExtendAppExtensionReferences` depends on `_ResolveAppExtensionReferences`. Optional, though.

Reply:
@jonathanpeppers would that still run `_ResolveAppExtensionReferences` even if `_ExtendAppExtensionReferences`'s Condition was false?

Reply:
Yes it will , the condition only applies to the current target, not its dependency chain.

Reply:
I had to test this!

It seems the dependency chain is not executed: https://gist.github.com/rolfbjarne/86fa3d32923fe7198549534ca00a850d

And the MSBuild documentation says: "If the condition evaluates to false, the target will not execute the body of the target or any targets that are set in the DependsOnTargets attribute." (https://docs.microsoft.com/en-us/visualstudio/msbuild/target-element-msbuild?view=vs-2019)

Reply:
I must have been thinking about Inputs/Outputs then.